tests/e2e/tests/01_EmbedStreamTest.js

```diff
@@ -0,0 +1,44 @@
+export default {
```

Comment:
And we are using `module.exports = ` everywhere in the server side code, hence it would make sense to me to stay consistent rather than mixing import syntaxes.